### ISSUE TITLE
Requesting for training logs

### ISSUE THREAD

**Issue by vateye** [OpenGVLab/UniFormerV2#21]
Hi, is it convenient for you to share the training logs for all of the experiments? I would be appericiated if you share those precious training logs as the reference for reproducing the results. Thanks!

**Comment by Andy1621** [comment]
Since the model code is recleaned before publishing, the output is different from the current codebase's, which may cause misunderstanding. 
But the loss and accuracy are the same. However, the logs are too many to be released. I can provide part of them. 
Could you please tell me the experiments you are conducting? Then I can find the corresponding log and upload them to the Google Drive.

**Comment by vateye** [comment]
Currently, I am reproducing the results for k400_b16_8frames,  k710_b16_8frames, and ssv2_b16_16frames. Would you mind sharing those logs as the reference for my reproducing? Thanks.

**Comment by Andy1621** [comment]
Thanks for your question, I have added the corresponding log [here](https://drive.google.com/drive/folders/1BBHg27giDz12K2bsYtK3A5tOBDX9AgnW?usp=share_link).

I tried to add `EMA` in the original experiments, but it only works sometimes. So I remove `EMA` in the later experiments.
For the logs of k400_b16_f8 and ssv2_b16_f16, they show the `EMA` testing, and the first one is testing without `EMA`.
For the logs of k710_b16_f8, I have tested the model on k400. The first one is k710, the second one is k400. Note that without extra fine-tuning on the k400, the top-1 accuracy is similar to training on only the k400.
